config.json.example

```diff
@@ -3,5 +3,9 @@
   "hidserv": {
     "hostname": "avgfxawdn6c3coe3.onion",
     "key": "Il8Xas7uf6rjtc0LxYwhrx"
-  }
+  },
+  "vmsizes": {
+        "sd_app": 10,
```

Comment:
I will update this.

Reply:
Be

> nit: is there a reason the underscore here is required? Intuitively it sounds like having the actual vm name would be less-error prone here (`sd-app`, `sd-log`)

Because Python variables can not have `-` in the name. This is used in the Jinja template to access the config.json values.

Errors which we will see otherwise:

```
    Set up dom0 config files, including RPC policies, and create VMs

    [ERROR   ] Rendering exception occurred: Jinja variable 'dict object' has no attribute 'sd_app'

    [CRITICAL] Rendering SLS 'base:sd-app' failed: Jinja variable 'dict object' has no attribute 'sd_app'

    [ERROR   ] Rendering exception occurred: Jinja variable 'dict object' has no attribute 'sd_log'

    [CRITICAL] Rendering SLS 'base:sd-log' failed: Jinja variable 'dict object' has no attribute 'sd_log'

    local:

        Data failed to compile:

    ----------

        Rendering SLS 'base:sd-app' failed: Jinja variable 'dict object' has no attribute 'sd_app'

    ----------

        Rendering SLS 'base:sd-log' failed: Jinja variable 'dict object' has no attribute 'sd_log'
```